Configurations/windows-makefile.tmpl

```diff
@@ -701,22 +703,22 @@ EOF
               if ($args{src} =~ /\.S$/) {
                    return <<"EOF";
 $target: "$args{generator}->[0]" $deps
-	set ASM=\$(AS)
-	$generator \$@.S
-	\$(CPP) $incs $cppflags $defs \$@.S > \$@.i && move /Y \$@.i \$@
+	cmd /C "set "ASM=\$(AS)" & $generator \$@.S"
```

Comment:
Again, the quotes in quotes don't look right...

Reply:
https://ss64.com/nt/cmd.html